src/content/docs/reference/example.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
Is this file necessary or can it be deleted?

Reply:
Removed.